### ISSUE TITLE
[Question]: replicate api issue

### ISSUE THREAD

**Issue by axz91** [run-llama/llama_index#8007]
### Question Validation

- [X] I have searched both the documentation and discord for an answer.

### Question

https://replicate.com/p/ae55hc3beupa7zvba6psrk7mei

when I do the index query using llama_index, only one fragment of document can be imported to the promt, what is the error

**Comment by axz91** [comment]
I just want load my vectorbase, and do the query,, no need Knowledge graph

**Comment by logan-markewich** [comment]
@axz91 what do you mean "only one fragment of document can be imported to the prompt" ?

**Comment by axz91** [comment]
> @axz91 what do you mean "only one fragment of document can be imported to the prompt" ?

yes... i embedding multiple nodes from multiple documents,  but only one chunk has sent to the llm for query
